VeryImportantComponent.js

```diff
@@ -1,14 +1,25 @@
-import { Alert } from '@nutanix-ui/prism-reactjs';
-import { useGetTranslation } from '../../hooks/useGetTranslation';
+import { Button } from 'antd';
+import { useCallback } from 'react';
+
+function VeryImportantComponent (props) {
+  const text = 'Select a number from 1 to 10';
+
+  if (!props.visible) {
+    return;
+  }
+
+  const clickButton = useCallback(() => {
+    // performs a very important function
+  }, []);
 
-function VeryImportantComponent () {
-  const { i18n__T } = useGetTranslation();
```

Comment:
`IN-HOUSE`: Use the callback function directly instead of using an arrow function